integrationTests/vm/txsFee/migrateDataTrie_test.go

```diff
@@ -274,7 +273,7 @@ func migrateDataTrie(
 	testContext.CleanIntermediateTransactions(t)
 
 	gasLocked := "00" //use all available gas
-	txData := core.BuiltInFunctionMigrateDataTrie + "@" + hex.EncodeToString([]byte{byte(core.NotSpecified)}) + "@" + hex.EncodeToString([]byte{byte(core.AutoBalanceEnabled)}) + "@" + gasLocked
+	txData := core.BuiltInFunctionMigrateDataTrie + "@" + gasLocked
```

Comment:
so we will automatically handle the migration parameters. This is safer for the user but it will be our job to always ensure that **any** previous trie node can and will be converted to the latest version.

Reply:
Indeed, when a new version will be implemented, we must test all cases and ensure that all previous versions can be migrated to the new version.

Reply:
... or migrate all tries :D